src/mitigation.c

```diff
@@ -54,18 +56,37 @@ void mitigation_add_resource(struct mitigation *m, const char *name, const char
 
 	r->resource = res;
 
-	list_append(&m->resources, &r->list_node);
+	// resources that have the attribute fallback will be added to a seperate list
+	if(fallback)
```

Comment:
Just so we're clear, I think that lack of the attribute should suffice for a false indication as well. That is:
- no attribute:  false
- "false": false
- "true": true